Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+final class Book
+{
+    /** @var string */
+    private $title;
+
+    /** @var int */
+    private $yearOfPublication;
+
+    public function __construct()
+    {
+    }
```

Comment:
Please let's drop the empty constructor. (It's not required, and if the parent class had/got a non-empty constructor, the parent constructor would be overwritten.)